docker-compose.yml

```diff
@@ -13,6 +13,10 @@ crossdock:
       - BEHAVIOR_DANCE=client,server,transport
       - BEHAVIOR_SING=client,server,transport
 
+      - REPORT=json
```

Comment:
This is really cool! I think what we'll want to do is support multiple reporters, so that you don't lose the console output:

```
- REPORT=list,json
```